contracts/src/core/interfaces/IEigenDAServiceManager.sol

```diff
@@ -4,10 +4,10 @@ pragma solidity ^0.8.9;
 import {IServiceManager} from "lib/eigenlayer-middleware/src/interfaces/IServiceManager.sol";
 import {BLSSignatureChecker} from "lib/eigenlayer-middleware/src/BLSSignatureChecker.sol";
 import {BN254} from "lib/eigenlayer-middleware/src/libraries/BN254.sol";
-import {IEigenDAThresholdRegistry} from "src/core/interfaces/IEigenDAThresholdRegistry.sol";
+import {IEigenDAThresholdRegistryBase} from "src/core/interfaces/IEigenDAThresholdRegistry.sol";
 import {EigenDATypesV1 as DATypesV1} from "src/core/libraries/v1/EigenDATypesV1.sol";
 
-interface IEigenDAServiceManager is IServiceManager, IEigenDAThresholdRegistry {
+interface IEigenDAServiceManager is IServiceManager, IEigenDAThresholdRegistryBase {
```

Comment:
prob good to document why this extends Base instead of ThresholdRegistry

Reply:
Will do

Reply:
https://github.com/Layr-Labs/eigenda/pull/1578/commits/ddf56dce74d3b8d4eeebb610bea3a7b152c37362